src/config/dynamic-theme-fixes.config

```diff
@@ -1561,6 +1561,13 @@ CSS
 
 ================================
 
+feynmanlectures.caltech.edu
+
+INVERT
+.figure > img
```

Comment:
`.figure > img[src$=".svgz"]`
Have a look how that looks. This will hopefull only target the math images.

Reply:
I checked about 25 chapters and it looks good.